posts/2020-10-29-macos-arm64.rst

```diff
@@ -123,7 +123,7 @@ You can rerender using,
 
    conda smithy rerender
 
-For python packages, add the following to ``recipe/meta.yaml``,
+For python packages, add one or more of the following to ``recipe/meta.yaml`` if they are used in `host:`,
```

Comment:
This should only apply to cython, numpy and pybind11

Reply:
Better now?

For python packages, add one or more of the following to ``recipe/meta.yaml`` as needed, noting that you *must* only add `numpy`, `cython`, and/or `pybind11` if  they are used in `host:` as well,